daisy/image.go

```diff
@@ -115,6 +117,23 @@ func (i *Image) MarshalJSON() ([]byte, error) {
 	return json.Marshal(*i)
 }
 
+type guestOsFeatures []string
+
+// UnmarshalJSON unmarshals GuestOsFeatures.
+func (g *guestOsFeatures) UnmarshalJSON(b []byte) error {
```

Comment:
We could leverage similar logic on Create Instances probably. Not needed in this PR obviously.